tests/unit/s2n_self_talk_client_hello_cb_test.c

```diff
@@ -144,6 +146,12 @@ int client_hello_swap_config(struct s2n_connection *conn, void *ctx)
         return -1;
     }
 
+    /* Verify correct server name is returned. */
+    received_server_name = s2n_get_server_name(conn);
+    if (received_server_name == NULL || strcmp(received_server_name, sent_server_name)) {
```

Comment:
EXPECT_NOT_NULL(received_server_name);
EXPECT_STRING_EQUAL(received_server_name, sent_server_name);

Reply:
It will cause compilation problem because EXPECT_NOT_NULL and EXPECT_STRING_EQUAL are not within BEGIN_TEST() and END_TEST() statements.

Reply:
good point - thanks